pkgs/tools/misc/rmate-sh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rmate";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "aurora";
+    repo = "rmate";
+    rev = "refs/tags/v${version}";
```

Comment:
Normally "v${version}" alone is fine.

Reply:
Personal preference, I'd like to know if upstream decides to replace the tag with a branch.